api/Console.json

```diff
@@ -231,19 +237,19 @@
               "version_added": "8.3.0"
             },
             "opera": {
-              "version_added": true
+              "version_added": "55"
```

Comment:
Can you add opera_android here as well, as you've done above? (I'll comment just "opera_android" for this in the following.)